src/components/policy/policy_regular/src/policy_table/types.cc

```diff
@@ -1003,7 +1003,41 @@ AppLevel::AppLevel(const Json::Value* value__)
 
 Json::Value AppLevel::ToJsonValue() const {
   Json::Value result__(Json::objectValue);
-  impl::WriteJsonField("count_of_TLS_errors", count_of_tls_errors, &result__);
+  impl::WriteJsonField("minutes_in_hmi_full", minutes_in_hmi_full, &result__);
+  impl::WriteJsonField("app_registration_language_gui",
+                       app_registration_language_gui,
+                       &result__);
+  impl::WriteJsonField("app_registration_language_vui",
+                       app_registration_language_vui,
+                       &result__);
+  impl::WriteJsonField(
+      "minutes_in_hmi_limited", minutes_in_hmi_limited, &result__);
+  impl::WriteJsonField(
+      "minutes_in_hmi_background", minutes_in_hmi_background, &result__);
+  impl::WriteJsonField(
+      "minutes_in_hmi_background", minutes_in_hmi_background, &result__);
+  impl::WriteJsonField("minutes_in_hmi_none", minutes_in_hmi_none, &result__);
```

Comment:
Was this just copied from the external policies types.cc? There is a duplicate `minutes_in_hmi_none`. Also just because these behavior exists in external policies, should it also exist in regular policies?